pdr_backend/analytics/predictoor_dashboard/predictoor_dash.py

```diff
@@ -24,6 +29,18 @@ def predictoor_dash(ppss: PPSS, debug_mode: bool):
     if not debug_mode:
         webbrowser.open(f"http://127.0.0.1:{port}/")
     folder = ppss.lake_ss.lake_dir
-    app.layout.children[0].data = folder
+    app.lake_dir = folder
+    try:
+        app.feeds_data = get_feeds_data_from_db(folder)
+        app.predictoors_data = get_predictoors_data_from_payouts(
+            get_user_payouts_stats_from_db(folder)
+        )
+    except Exception as e:
+        print(
```

Comment:
Shouldn't we use the `logger` here?

Reply:
no, it's a CLI print when running the command, not a debugging print.